ci/Debian11Container.dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM node:lts-bullseye
+WORKDIR /
+SHELL ["/bin/bash", "-c"]
+
+# Install emscripten
+RUN git clone https://github.com/emscripten-core/emsdk.git \
+    && emsdk/emsdk install 3.1.59 \
+    && emsdk/emsdk activate 3.1.59 \
+    && source emsdk/emsdk_env.sh
+ENV PATH="$PATH:/emsdk/upstream/bin:/emsdk/upstream/emscripten"
+
+# Install FPM
```

Comment:
to package webapp deb installers

Reply:
I noticed in the next commit the file is renamed into desktop specific one, I guess we don't need FPM then?

Reply:
> I noticed in the next commit the file is renamed into desktop specific one, I guess we don't need FPM then?

It is still needed to bundle the built into a deb according to my understanding. We still need to `FPM` is used in `Publish.js` which is called during the 'Build deb and publish' stage.

Reply:
Right! thank you, I forgot about that part